contracts/StakingPoolV2.sol

```diff
@@ -31,43 +30,41 @@ contract StakingPoolV2 is IStakingPoolV2, StakedElyfiToken {
     mapping(address => uint256) userIndex;
     mapping(address => uint256) userReward;
     mapping(address => uint256) userPrincipal;
+    IERC20 rewardAsset;
+    bool isPause;
```

Comment:
How about renaming `isPause` to `paused` and `isNotPause` to `whenNotPaused`?
https://github.com/aave/protocol-v2/blob/61c2273a992f655c6d3e7d716a0c2f1b97a55a92/contracts/protocol/lendingpool/LendingPoolStorage.sol#L25